mirgecom/fluid.py

```diff
@@ -273,12 +279,13 @@ def _aux_shape(ary, leading_shape):
         return ()
 
 
+# Still needed?
 def get_num_species(dim, q):
     """Return number of mixture species."""
     return len(q) - (dim + 2)
 
 
-def split_conserved(dim, q):
+def cv_from_flat(dim, q):
```

Comment:
Quite possibly, we should deprecate rather than hard-remove these functions, to avoid breaking other code built on this.